meeting-minutes/team-minutes/nov2-team-minutes.md

```diff
@@ -5,17 +5,57 @@
 - [Eric] Need a new requirement to work on.
 - [Eric] Explanation of #155. CI has been implemented before I was assigned to the issue #155
 - [Eric] Setup DB.
-- [Eric] fix 'Unexpected end of JSON input'
+- [Eric] Fix 'Unexpected end of JSON input'
 - [Aleks] Should we use console to pull/commit or VS to pull/commit
 - [Aleks] Does it matter if we use VS or console to pull/commit
 - [Aleks] On the form for front end we have a "submit" button but we also have a "back" and "next" button, what is the purpose of the "back" and "next" buttons?
 - [JP] Extraneous files in top-level src
 - [JP] Explain ESLint process
 - [Katrina] Discuss possible ways to obtain user availability (the current way is a field for the day of a week, and pre-determined time ranges in a drop-down to choose from. A possible alternative could be something like this: https://fullcalendar.io/)
 - [Katrina] The course website (https://people.ok.ubc.ca/bowenhui/499/) outlines suggested bullet points for November 6th (Friday). We should discuss how to tackle this (possibly, since heuristics evaluations deal with the flow/how users interact with the platform (primarily concerned with the client side)), the front-end team members could work on that.
+- 
 
 ## Detailed Notes:
+- Every meeting we are going to do a status update meeting
+- Status meeting is where everyone tells what they have been working on during the sprint and if they had any problems
+- Delete the time availability for the front end form
+- Database is setup
+- Basic front end is setup
+- Code will automatically run ESLint
+- ESLint will keep all syntax the same 
+- Run ESLint before you do a start the program so that your syntax is the same as the standard we are using

```

Comment:
Sorry, just to clarify: by this, do you mean run EsLint locally before we commit/push to Github?

Reply:
Yes, I will fix it right now to make it more clear